### PR TITLE
[14.0] [mig] hr_timesheet_sheet_autodraft

### PR DESCRIPTION
**Pull request by dsolanki-initos**
Depends on PR #434

### CODE REVIEW

**Comment by dsolanki-initos** [comment]
@alexey-pelykh If possible can you please review the PR?

File: hr_timesheet_sheet_autodraft/tests/test_hr_timesheet_sheet_autodraft.py
Comment (alexey-pelykh) [inline hr_timesheet_sheet_autodraft/tests/test_hr_timesheet_sheet_autodraft.py]: But that should be the default, no?

Comment (dsolanki-initos) [reply hr_timesheet_sheet_autodraft/tests/test_hr_timesheet_sheet_autodraft.py]: For that field default value is not set.

Comment (alexey-pelykh) [reply hr_timesheet_sheet_autodraft/tests/test_hr_timesheet_sheet_autodraft.py]: Well, that's the point of `test_no_autocreate_by_default` test? That by default, w/o setting the value, it's not autocreated?

Comment (dsolanki-initos) [reply hr_timesheet_sheet_autodraft/tests/test_hr_timesheet_sheet_autodraft.py]: It can be created i'll remove it.

**Review by alexey-pelykh — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
@dsolanki-initos please follow the migration guidelines https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0 regarding that version change should be in the [MIG] commit, yet I'm not blocking the PR due to that this time

**Review by fshah-initos — APPROVED** [review]
LGTM. Functional tests locally and code Review.

**Pull request merged**
